### PR TITLE
KEYCLOAK-11417 Internal server error on front channel logout with exp…

### PR DESCRIPTION
**Pull request by martin-kanis**
…ired session

I also fixed debug messages in CookieHelper.java which didn't show provided parameters properly.

### CODE REVIEW

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/LogoutTest.java [hunk 1]
```diff
@@ -118,6 +134,37 @@ public void logoutSession() {
...
+
+            // wait for a timeout
+            // setTimeOffset doesn't work because session cookie is not invalidated thus the logout flow would continue with browser logout
+            TimeUnit.SECONDS.sleep(3);
+
+            String logoutUrl = oauth.getLogoutUrl().redirectUri(oauth.APP_AUTH_ROOT).idTokenHint(idTokenString).build();
+            driver.navigate().to(logoutUrl);
+
+            // should not throw an internal server error
+            assertTrue(appPage.isCurrent());
```
Comment (hmlnarik) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/LogoutTest.java:159]: Nit: should be 
```suggestion
            appPage.assertCurrent();
```
to have nicer failure report.

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/LogoutTest.java [hunk 2]
```diff
@@ -118,6 +134,37 @@ public void logoutSession() {
...
+
+            // check if the back channel logout succeeded
+            driver.navigate().to(oauth.getLoginFormUrl());
+            WaitUtils.waitForPageToLoad();
+            assertTrue(loginPage.isCurrent());
```
Comment (hmlnarik) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/LogoutTest.java:164]: Same as above

**Review by hmlnarik — COMMENTED** [review]
Looks good by me but I'd like @mposolda to have proper look.

**Review by mposolda — APPROVED** [review]
@martin-kanis Looks good to me, Thanks for the fix!

**Pull request merged**